djlsp/parser.py

```diff
@@ -1,31 +1,42 @@
+import logging
 import re
+from functools import cached_property
 from re import Match
```

Comment:
Als we `re` al geimport hebben waarom importen we `Match` dan nog apart ipv gewoon `re.Match` aan te roepen?

Reply:
Used Match for type hints, I like that without namespace